frontend/src/v5/store/containers/containers.helpers.ts

```diff
@@ -29,6 +29,17 @@ export const filterContainers = (federations: IContainer[], filterQuery: string)
 	) => [name, code, type].join('').toLowerCase().includes(filterQuery.trim().toLowerCase()))
 );
 
+export const isBusyInBackend = (status: UploadStatuses) => {
+	const busyInBackend = [
```

Comment:
It would be better to do the reverse, i.e. !(undefined || ok || failed)

This is because we may add more processing stages in the future (or rename them), there will be less things to change.

Reply:
Make sense!